index.js

```diff
@@ -49,13 +51,25 @@ const sendResponse = ( req, settings, fn ) => {
 		}
 
 		let { body, headers, statusCode } = response;
+		const contentType = req.xhr
+			? req.xhr.getResponseHeader( 'Content-Type' )
+			: headers[ 'content-type' ];
+
+		if ( ! req.xhr ) {
+			// node
+			if ( shouldProcessInStreamMode( contentType ) ) {
+				const error = new Error( 'stream mode processing is not yet implemented for Node.js' );
+				return fn( error, body, headers );
```

Comment:
can/should we pass body rather than null here, given that it’s available to us?

Reply:
Yep that's fine, `body` can help with debugging.